pkg/storage/db.go

```diff
@@ -240,6 +256,22 @@ func (db *DB) GetIPAddressesForHostname(ctx context.Context, hostname string) ([
 	return db.runQuery(ctx, sqlstmt, hostname)
 }
 
+// GetAssetsByHostnameAtTime gets the assets who have hostname at the specified time
+func (db *DB) GetAssetsByHostnameAtTime(ctx context.Context, hostname string, at time.Time) ([]domain.NetworkChangeEvent, error) {
+	// ok to hardcode this timestamp as the service in which this code runs went into production sometime after this date:
+	from, _ := time.Parse(time.RFC3339, "2019-04-22T00:00:00+00:00")
```

Comment:
If we stick with a strategy like this, won't we be bound to always scan all partitions of a table between the epoch date and the search date? It seems like that would minimize the usefulness of a time based partition.

Reply:
It's a good question.  I suppose I could use `max(ts)`..  I'll think about it and try to re-work this strategy.